.travis.yml

```diff
@@ -39,6 +39,10 @@ env:
       OS_VERSION=31
       PYTHON_VERSION=3
       ENGINE=docker
+    - ACTION=markdownlint
+      CONTAINER_NAME=ruby
```

Comment:
I don't think that container name should be defined here. It may break local testing due conflicting names